src/fileSystemActions.js

```diff
@@ -1,21 +1,21 @@
 const nativeUI = require("./nativeUI");
+const util = require("util");
 const fs = require("fs");
 
 const saveToFile = content => {
-  const fileName = nativeUI.getFileNameFromUser();
-  if (fileName === undefined) {
-    return;
-  }
-  fs.writeFile(fileName, content, error => {
-    if (error) {
-      nativeUI.displayErrorMessage("An error has occurred", error.message);
-    } else {
-      nativeUI.displayInfoMessage(
-        "File Saved",
-        "Your document has been saved."
-      );
-    }
-  });
+  const writeFile = util.promisify(fs.writeFile);
+
+  nativeUI
```

Comment:
I'm trying to test this promise chain.

- In my test, `nativeUI.getFileNameFromUser()` is stubbed to return a resolved promise.
- Then I attempt to stub the `util.promisify`ed `fs.writeFile()` to also return a resolved promise.
- Then I expect that the stub of `nativeUI.displayInfoMessage()` is called

Reply:
Missing `return` statement... 😱